libs/model/__tests__/community/stake-lifecycle.spec.ts

```diff
@@ -127,7 +130,7 @@ describe('Stake lifecycle', () => {
       .to.eventually.be.rejected;
   });
 
-  it('should set and get community stake', async () => {
+  it.skip('should set and get community stake', async () => {
```

Comment:
should we review this skip tomorrow?

Reply:
Fixed it just now... another discrepancy between CommunityStakes Sequelize model and production-DB/migration